app/assets/stylesheets/blacklight.scss

```diff
@@ -37,3 +37,30 @@
   @extend .col-md-6;
   @extend .col-sm-12;
 }
+
+.documents-list {
+  .document {
+    position: relative;
+    padding-left: 140px;
+  }
+
+  .index_title {
+    font-size: $font-size-large;
+  }
+
+  .document-counter {
+    display: none;
+  }
+
+  .document-thumbnail {
```

Comment:
Is there a better way to position the image there (without changing the markup..)? Other things I tried were foiled by the `<dl>` `clear: left` styling.